app/src/views/Dashboard.vue

```diff
@@ -203,26 +203,15 @@
       class="info-dialog"
       width="80vw"
     >
-      <template v-slot:activator="{ on, attrs }">
-        <v-btn
-          color="red lighten-2"
-          dark
-          v-bind="attrs"
-          v-on="on"
-        >
-          Click Me
-        </v-btn>
-      </template>
-
-      <template style="background: #FFF">
+      <template>
         <div
           class="d-flex justify-between px-7 py-4"
           width="100%"
-          style="background: #CED9E0; justify-content: space-between; align-items: center;"
+          style="justify-content: space-between; align-items: center;"
+          :style="{background: this.$vuetify.theme.dark ? 'var(--v-grey-darken4)' : '#CED9E0'}"
         >
-          <span class="font-medium text-h6">
-            <!-- Convert the first letter to uppercase. -->
-            {{ showText[0].toUpperCase() + showText.substring(1) }}
```

Comment:
good riddance! nice that Vuetify now supports text-capitalize :-) but also here, probably only `{{ showText }}` is enough